third_party/libwebp/src/enc/predictor_enc.c

```diff
@@ -16,11 +16,8 @@
 
 #if defined(STARBOARD)
 #include "starboard/client_porting/poem/assert_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
 #endif
 
-#include <stdlib.h>
```

Comment:
This was added by commit 99819a64.
I probably shouldn't have removed it in _this_ PR. 🤷
There's plenty more similar stuff elsewhere.